lib/src/streams/stream_handler.dart

```diff
@@ -384,9 +384,9 @@ class StreamHandler extends Object with TerminatableMixin, ClosableMixin {
     }
 
     if (msg is DataStreamMessage) {
-      _sendData(stream, msg.bytes, endStream: false);
+      _sendData(stream, msg.bytes, endStream: msg.endStream);
```

Comment:
Absolutely, yes. I don't think I'll use it in gRPC, since I need to wait for the stream to be closed anyway, but it might be useful.